dev-libs/libpreludedb/libpreludedb-5.1.0.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+USE_RUBY="ruby24 ruby25 ruby26"
+DISTUTILS_OPTIONAL=1
+
+inherit autotools distutils-r1 ruby-single
+
+DESCRIPTION="Framework to easy access to the Prelude database"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
```

Comment:
Does the SONAME change? I see https://github.com/Prelude-SIEM/libpreludedb/compare/4.1.0...5.1.0#diff-67e997bcfdac55191033d57a16d1408aL32 but I'm not sure. If it does, we should add a subslot so that the package manager recompiles the packages that depend on libpreludedb to prevent ABI breakage.

Unless the version range specifier ensure that no ABI breakage could happen due to package upgrades ...

Reply:
If I well understood, I fixed this. Let me know if I did the right thing

For libprelude, current version is 28 and age is 0